analysis.go

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) nexB Inc. and others. All rights reserved.
+// ScanCode is a trademark of nexB Inc.
+// SPDX-License-Identifier: Apache-2.0
+// See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+// See https://github.com/nexB/go-inspector for support or download.
+// See https://aboutcode.org for more information about nexB OSS projects.
+//
+
+package go_inspector
+
+import (
+	"debug/buildinfo"
+	"encoding/json"
+)
+
+type Package struct {
+	Path    string `json:"path"`
+	Version string `json:"version"`
+	Sum     string `json:"sum"`
+}
+
+// InspectLibraries return all 3rd party packages used by the binary
+func InspectLibraries(filepath string) ([]Package, error) {
+	var packages []Package
+	bi, _ := buildinfo.ReadFile(filepath)
+	for _, v := range bi.Deps {
+		item := Package{Path: v.Path, Version: v.Version, Sum: v.Sum}
+		packages = append(packages, item)
+	}
+	return packages, nil
+}
+
+// ConvertToJSONWithPosixPaths allows to inspect libraries at the specified path, convert the file paths to POSIX format, and obtain the JSON representation of the library information with POSIX paths.
+func ConvertToJSONWithPosixPaths(path string) ([]byte, error) {
```

Comment:
You need to write a test that asserts that we get exactly what we expect from running this on a test go binary.